runtime/compiler/control/J9Options.cpp

```diff
@@ -1425,6 +1425,17 @@ J9::Options::fePreProcess(void * base)
          }
       }
 
+   // -XX:+PrintCodeCache will be parsed twice into both AOT and JIT options here.
+   const char *xxPrintCodeCacheOption = "-XX:+PrintCodeCache";
+   const char *xxDisablePrintCodeCacheOption = "-XX:-PrintCodeCache";
+   int32_t xxPrintCodeCacheArgIndex = FIND_ARG_IN_VMARGS(STARTSWITH_MATCH, xxPrintCodeCacheOption, 0);
```

Comment:
`STARTSWITH_MATCH`  should be replaced with `EXACT_MATCH` because somethink like `-XX:+PrintCodeCachemmm` should not be accepted